README.md

```diff
@@ -18,3 +18,26 @@ To use these files in your own project:
 8. Repeat steps 3 through 7 for any other files you wish to use in your project
 9. Back in UEFN click on **Verse** and choose **Build Verse Code**
 10. If everything compiles correctly then the functions, classes and structs defined in the files you added should now be available to your own devices and code.
+
+## Using Array
+
+1. Add `array.verse` to the root of your Verse project
+2. Import the Array functions by putting `using { Array }` at the top of your device Verse file
```

Comment:
I thought that anything in the main directory of a project was auto imported/available. Is that not true?

Or is this because you put them in a module?

Reply:
I'm gonna assume this is the case. I'll give it a try later and submit a PR if not.